Security preview releases

Ganwtrs opened the pull request:
Partially addresses #1265 , but I don’t know the answer to what @An-anonymous-coder proposed

> It should also specify, for example, if the [2025092700 stable SP] release is prioritized above the [2025100300 beta non-SP] release, or if that is considered a downgrade.

thestinger commented:
You can rebase it here without opening a new pull request if you want to do it that way, which is what I would prefer, but I assume the reason it's this way is because you're using the GitHub web interface which lacks support for making a pull request in a form that can be accepted i.e. not with a messy history.

Ganwtrs committed:
feat: add mention of security preview releases

thestinger commented:
@Ganwtrs Can you just reopen it? It looks like the change got dropped.

Ganwtrs committed:
Merge branch 'GrapheneOS:main' into Security-preview-releases

Ganwtrs committed:
Remove comparison with other OEMs in features page

This is in case it might be outdated, for example other OEMs are applying some patches but not all of them
https://xcancel.com/GrapheneOS/status/1988448504029630886#m

Ganwtrs committed:
Remove comparison with other OEMs in usage page
